portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,17 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about me to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['I have a scar on the back of my head from when I fell backwards out of a chair when I was in 5th grade.', 'I backpacked through Portugal with a Catalan Troop.', 
```

Comment:
I'd recommend putting each of the list entries on its own line for ease of reading. Good general practice, except for lists with very short elements like numbers.

Reply:
Sounds good, made the change.